src/picongpu/include/particles/ionization/byField/BSI/BSI.def

```diff
@@ -20,24 +20,37 @@
 
 #pragma once
 
+#include "types.h"
+
 namespace picongpu
 {
 namespace particles
 {
 namespace ionization
 {
-
-    /** \struct BSI
-     * \brief Barrier Suppression Ionization
+    /** \struct BSI_Impl
+     * \brief Barrier Suppression Ionization - Implementation
      *
      * - takes the ionization energies of the various charge states of ions
```

Comment:
Please move this documentation to `BSI` because the user used the class `BSI` and not `BSI_Impl`.
`BSI_Impl` is only used internel (from the developer <- you )